src/argilla/training/base.py

```diff
@@ -234,6 +236,12 @@ def __init__(
             )
 
         self._logger.info(self)
+        self._track_trainer_usage(framework=framework, task=self._rg_dataset_type._RECORD_TYPE.__name__)
+
+    def _track_trainer_usage(self, framework: str, task: str):
+        if not self._CLIENT:
```

Comment:
I think this is not necessary since the telemetry client will be always initialized: See https://github.com/argilla-io/argilla/blob/6fb45b3a70ccd511b5e4b8305af212efb6cc6df3/src/argilla/utils/telemetry.py#L89C1-L89C8